modules/api-resources/api-resources-full/pom.xml.versionsBackup

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We need to remove this file.

Reply:
I have added it to git ignore. Updating the changes now.

Reply:
PR automatically closed with head reset. Changes in new PR.